src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -136,6 +133,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DefaultAppHostRuntimeIdentifier Condition="$(DefaultAppHostRuntimeIdentifier.StartsWith('win')) and '$(PlatformTarget)' == 'x86'">win-x86</DefaultAppHostRuntimeIdentifier>
     <DefaultAppHostRuntimeIdentifier Condition="$(DefaultAppHostRuntimeIdentifier.StartsWith('win')) and '$(PlatformTarget)' == 'ARM'">win-arm</DefaultAppHostRuntimeIdentifier>
     <DefaultAppHostRuntimeIdentifier Condition="$(DefaultAppHostRuntimeIdentifier.StartsWith('win')) and '$(PlatformTarget)' == 'ARM64'">win-arm64</DefaultAppHostRuntimeIdentifier>
+    <DefaultAppHostRuntimeIdentifier Condition="$(NETCoreSdkRuntimeIdentifier.EndsWith('arm64')) and
```

Comment:
Do we need `EndsWith('arm64')`? Isn't it good enough to say `osx`+`<6.0`?

And why use `NETCoreSdkRuntimeIdentifier` instead of `DefaultAppHostRuntimeIdentifier`?

Reply:
Changed to use `DefaultAppHostRuntimeIdentifier` instead of `NETCoreSdkRuntimeIdentifier`. I don't think we need the `Ends...` here, it's enough to check for `osx & < 6.0`, but left it here to make things clearer (since we are replacing that same string later).